storage/aptosdb/src/db_options.rs

```diff
@@ -155,6 +155,7 @@ where
         let mut cf_opts = Options::default();
         cf_opts.set_compression_type(DBCompressionType::Lz4);
         cf_opts.set_block_based_table_factory(&table_options);
+        cf_opts.add_compact_on_deletion_collector_factory(0, 0, 0.4);
```

Comment:
What does this do? How does this save disk space?

Reply:
It makes rocksdb more aggressively remove old data made stale vlby deletions.  (And see the pr being picked)